online-inference/tensorizer-isvc/tensorizer_hf_isvc/load_model.py

```diff
@@ -0,0 +1,60 @@
+import time
+
+import torch
+from tensorizer import TensorDeserializer
+from tensorizer.utils import convert_bytes, get_mem_usage, no_init_or_tensor
+from transformers import AutoConfig, AutoModelForCausalLM, GPTJForCausalLM
+
+# GLOBALS
+
+DEVICE = "cuda"
+
+"""
+Loads the model using Tensorizer or Huggingface.
+
+params:
+- model_load_type (str): Type of model load [Options: "tensorizer", "hf"]
+"""
+
+
+def load_model_based_on_type(model_load_type: str = "tensorizer"):
```

Comment:
This function sits on its own, so can we add a `path` argument that we can default to `/mnt/pvc`?